HISTORY.md

```diff
@@ -17,6 +17,7 @@
 * Fixed an assertion failure in CompactionIterator when write-prepared transaction is used. We prove that certain operations can lead to a Delete being followed by a SingleDelete (same user key). We can drop the SingleDelete.
 * Fixed a bug of timestamp-based GC which can cause all versions of a key under full_history_ts_low to be dropped. This bug will be triggered when some of the ikeys' timestamps are lower than full_history_ts_low, while others are newer.
 * Explicitly check for and disallow the `BlockBasedTableOptions` if insertion into one of {`block_cache`, `block_cache_compressed`, `persistent_cache`} can show up in another of these. (RocksDB expects to be able to use the same key for different physical data among tiers.)
+* Users who configured a dedicated thread pool for bottommost compactions by explicitly adding threads to the `Env::Priority::BOTTOM` pool will no longer see RocksDB schedule automatic compactions exceeding the DB's compaction concurrency limit (determined by `max_background_compactions` or `max_background_jobs` when compaction pressure exists, or one when no pressure exists).
```

Comment:
`s/one when no pressure/when no pressure`

Reply:
I actually mean the concurrency limit is one. https://github.com/facebook/rocksdb/blob/937fbcbddca3d1a777271feb7dcce850d3bab86f/db/db_impl/db_impl_compaction_flush.cc#L2416-L2417

Agreed I phrased it oddly, and it's unnecessary to explain here anyway. Let me just refer the reader to `max_background_compactions` / `max_background_jobs` docs.

Reply:
Done.